doc/cephadm/host-management.rst

```diff
@@ -104,7 +104,12 @@ are free form and have no particular meaning by itself and each host
 can have multiple labels. They can be used to specify placement
 of daemons. See :ref:`orch-placement-by-labels`
 
-To add a label, run::
+Labels can be added when adding a host with the ``--labels`` flag::
+
+  ceph orch host add my_hostname --labels=my_label1
+  ceph orch host add my_hostname --labels=my_label1,my_label2
```

Comment:
```suggestion
  ceph orch host add my_hostname --labels=my_label1 --labels=my_label2
```

should work as well. 

I'm not yet sure if I should be super happy, if we introduce a new way to pass multiple labels. I think a better way would be to extend ceph_argparse to also support comma-separated lists of elements for `n=N` arguments. But, understandably, no one likes touching ceph_argparse.

Reply:
yeah